cmd/root/results.go

```diff
@@ -27,28 +28,105 @@ func requiredRunes(s string) []rune {
 	return arr
 }
 
-func wpm(events []event) float64 {
+// Remove backspaces from a list of events.
+//
+// Removing backspace events simplifies some calculations,
+// including wpm,
+func removeBackspaces(events []event) []event {
+	enb := []event{}
+	for _, e := range events {
+		if e.typed != "backspace" {
+			enb = append(enb, e)
+		}
+	}
+	return enb
+}
+
+// Calculates the words per minute (wpm) using the events in the list.
+// Also allows the duration to be overridden, which is useful
+// for calculating wpm per second, which is used in the `wpmGraph` function.
```

Comment:
Note that you should avoid using `wpmBase` and use the other `wpm` functions instead